added donate and event page forms, event page

prash240303 opened the pull request.

prash240303 committed:
added donation page

prash240303 committed:
added donate , registration forms and event page

prash240303 committed:
added the player

ravinder-chadha reviewed (changes requested):
Please do the following:
- Youtube Videos Allignment
- Volunteer and Patron Button gaps
- Exchange the forms of Registration, Volunteer for the event
- Sub title of volunteer form is to be fixed
- Fix the width of text area for asking reason
- Capitalize the P of past events
- Add padding or margin to Donate Now button in CTA
- Past Events, All attempts in single column in mobile view
- Width of podcast full screen such that it looks good
- Anonymous Feedback Text Container is to be added in upcoming events
- Read Articles page with a back to community button and heading on top

prash240303 committed:
Youtube Videos Allignment
Volunteer and Patron Button gaps
Exchange the forms of Registration, Volunteer for the event
Sub title of volunteer form is to be fixed
Fix the width of text area for asking reason
Capitalize the P of past events
Add padding or margin to Donate Now button in CTA
Past Events, All attempts in single column in mobile view
Width of podcast full screen such that it looks good
Anonymous Feedback Text Container is to be added in upcoming events

ravinder-chadha reviewed (approved):
Thank you! These look perfect.
